### DISCUSSION TITLE
Lightning Talks sign ups

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omigroup#19] [General]
Have a topic you'd like to share with the group? Pitch your topic below, and we will get you added to the list! 

Let's lift each other up, and learn together!

**Comment by lalalune** [comment]
What is an NFT (I mean, actually, like in code) and how can we use them for interoperability? Contract deployment, gas, proof of work vs proof of stake, WTF is going on with Eth 2.0?, on-chain vs off-chain and resources to deploy your own NFTs.

**Comment by robertlong** [comment]
I could do one on the work I did for Hubs on the glTF component data extension. This brings common object behaviors to glTF.
